packages/backend/src/services/product.ts

```diff
@@ -83,4 +84,21 @@ export class ProductService {
 
     return productId;
   }
+
+  public async buy({ userId, productId }: AddProductToFavorites) {
```

Comment:
Please add a check whether user is validated

Reply:
you can add his field in auth middleware and check in items/buy router

Reply:
![image](https://user-images.githubusercontent.com/92884408/187429196-d835d56c-7a69-4e42-9857-461b519156ae.png)
there is auth middleware in router